rest-api/src/main/java/com/cloudbees/workflow/flownode/FlowNodeUtil.java

```diff
@@ -73,9 +65,7 @@ private FlowNodeUtil() {
     }
 
     public abstract static class CacheExtensionPoint implements ExtensionPoint {
-        public abstract Cache<String,List<FlowNode>> getExecutionCache();
```

Comment:
I think this is fine, though it perhaps should have been `@Restricted(NoExternalUse.class)` from the start.

(Is there _any_ part of this plugin intended to be used as a Java API? I know we make some vague promises about supporting the REST API for other uses. Probably there is some SPI component used to allow proprietary metadata to be used from the UI?)